Misc/NEWS.d/next/Library/2019-06-14-08-30-16.bpo-19865.FRGH4I.rst

```diff
@@ -0,0 +1,2 @@
+:func:`ctypes.create_unicode_buffer()` no longer fails on non-BMP strings on
+Windows.
```

Comment:
According to my notes, AIX also use 16-bit wchar_t:
https://unicodebook.readthedocs.io/programming_languages.html#character-api-wchar-t

You might express it in a more positive way :-)

:func:`ctypes.create_unicode_buffer` now also supports non-BMP characters (outside [U+0000; U+FFFF] range) on platforms using 16-bit :c:type:`wchar_t` (ex: Windows, AIX).